UserInterface/FrameDemos/README.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Think this file should clarify that the Frame is a layout, rather than a view. Hence it doesn't wrap another view, as it isn't a view in the first place.